.codeclimate.yml

```diff
@@ -0,0 +1,47 @@
+prepare:
```

Comment:
Is this to be used locally or remotely? 

I don't think codeclimate is properly configured for this repo.

Reply:
This repo also does not have CoffeeScript files or SCSS files so we probably should modify this.

Reply:
@jackchoi @ascoppa i think it's better now